src/locales/ko/mystery-encounters/the-pokemon-salesman-dialogue.json

```
@@ -1,23 +1,23 @@
 {
-  "intro": "A chipper elderly man approaches you.",
-  "speaker": "Gentleman",
-  "intro_dialogue": "Hello there! Have I got a deal just for YOU!",
-  "title": "The Pokémon Salesman",
-  "description": "\"This {{purchasePokemon}} is extremely unique and carries an ability not normally found in its species! I'll let you have this swell {{purchasePokemon}} for just {{price, money}}!\"\n\n\"What do you say?\"",
-  "description_shiny": "\"This {{purchasePokemon}} is extremely unique and has a pigment not normally found in its species! I'll let you have this swell {{purchasePokemon}} for just {{price, money}}!\"\n\n\"What do you say?\"",
-  "query": "What will you do?",
+  "intro": "건장한 노신사가 다가왔다.",
+  "speaker": "신사",
+  "intro_dialogue": "거기, 안녕하신가! 자네에게 좋은 거래를 제안하지!",
+  "title": "포켓몬 상인",
+  "description": "\"이 {{purchasePokemon}}[[는]] 매우 독특하며 같은 종에서는 쉽게 볼 수 없는 특성을 갖고 있지! 나는 이 좋은 {{purchasePokemon}}[[를]] 자네에게 {{price, money}}에 보내고 싶다네!\"\n\n\"어떻게 생각하시오?\"",
+  "description_shiny": "\"이 {{purchasePokemon}} 매우 독특하며 같은 종에서는 쉽게 볼 수 없는 색을 갖고 있지! 나는 이 좋은 {{purchasePokemon}}[[를]] 자네에게 {{price, money}}에 보내고 싶다네!\"\n\n\"어떻게 생각하시오?\"",
```

```suggestion
  "description": "\"이 {{purchasePokemon}}[[는]] 매우 독특하며 같은 종에서는 쉽게 볼 수 없는 특성을 갖고 있지! 나는 이 좋은 {{purchasePokemon}}[[를]] 자네에게 {{price, money}}에 보내고 싶다네!\"\n\n\"어떻게 생각하는가?\"",
  "description_shiny": "\"이 {{purchasePokemon}} 매우 독특하며 같은 종에서는 쉽게 볼 수 없는 색을 갖고 있지! 나는 이 좋은 {{purchasePokemon}}[[를]] 자네에게 {{price, money}}에 보내고 싶다네!\"\n\n\"어떻게 생각하는가?\"",
```
앞 문장은 하게체고 뒷 문장은 하오체라서 하게체로 통일했습니다.